src/transformers/models/auto/processing_auto.py

```diff
@@ -145,6 +146,9 @@ def from_pretrained(cls, pretrained_model_name_or_path, **kwargs):
             key: kwargs[key] for key in ["revision", "use_auth_token", "local_files_only"] if key in kwargs
         }
         model_files = get_list_of_files(pretrained_model_name_or_path, **get_list_of_files_kwargs)
+        # strip to file name
+        model_files = [f.split("/")[-1] for f in model_files]
```

Comment:
Shouldn't that be handled in `get_list_of_files`?

Reply:
yeah I thought so too in the beginning, but think it's cleaner to get the actual file names from this function instead of trimming to the last file. What do you think? cc @sgugger as well?

Reply:
It should be done at the model_files level to be consistent with the distant repos.